### PR TITLE
feature/BLOC-146&144

### PR DESCRIPTION
**Pull request by Dmitrijg24k**

### CODE REVIEW

File: migrations/init_migration.up.sql [hunk 1]
```diff
@@ -30,7 +31,8 @@ CREATE TABLE IF NOT EXISTS community (
   tagsCount SMALLINT DEFAULT 0,
   followingUsers INT DEFAULT 0,
   ipfsHash VARCHAR(66),
-  ipfsHash2 VARCHAR(66)
+  ipfsHash2 VARCHAR(66),
+  network VARCHAR(20)
```
Comment (sergeyi23) [inline migrations/init_migration.up.sql]: Network needs to be `networkId` of type  INT. It is enough to store just the network id.

There should be another table, `Networks` with fields id and name. Then add foreign key from `networkId` column to `network` table.

File: migrations/init_migration.up.sql [hunk 2]
```diff
@@ -213,6 +215,7 @@ CREATE TABLE IF NOT EXISTS history (
   eventName VARCHAR(15),
   actionUser VARCHAR(66) NOT NULL,
   timeStamp INT NOT NULL,
+  network VARCHAR(20),
```
Comment (sergeyi23) [inline migrations/init_migration.up.sql]: The same as for community table.

File: src/controllers/evm-blockchain-controller.ts [hunk 3]
```diff
@@ -0,0 +1,348 @@
...
+const configRepository = new ConfigRepository(connDynamoDB);
+
+export async function readEvents(
+  readEventsRequest: ReadNotificationsRequestModel
+): Promise<ReadNotificationsResponseModel> {
+  try {
+    const Queues = {
+      [Network.Polygon]: POLYGON_INDEXING_QUEUE,
+      [Network.Edgeware]: EDGEWARE_INDEXING_QUEUE,
+      [Network.Sui]: SUI_INDEXING_QUEUE,
```
Comment (sergeyi23) [inline src/controllers/evm-blockchain-controller.ts]: SUI is not evm. Not needed here.

File: src/controllers/evm-blockchain-controller.ts [hunk 4]
```diff
@@ -0,0 +1,348 @@
...
+    };
+    const startsBlocks = {
+      [Network.Polygon]: process.env.POLYGON_START_BLOCK_NUMBER,
+      [Network.Edgeware]: process.env.EDGEWARE_START_BLOCK_NUMBER,
+      [Network.Sui]: '0',
+    };
```
Comment (sergeyi23) [inline src/controllers/evm-blockchain-controller.ts:139]: SUI is not evm. Not needed here.

File: src/core/blockchain/data-loader.ts [hunk 5]
```diff
@@ -166,22 +183,24 @@ export async function getAchievementsNFTConfig(
...
+  return contract.getDocumentationTree(communityId);
 }
 
 export async function getItemProperty(
   propertyId: number,
-  postId: number,
-  replyId?: number,
-  commentId?: number
+  postId: string,
+  network: Network,
```
Comment (sergeyi23) [inline src/core/blockchain/data-loader.ts:197]: May be better to make it the last argument for consistency.

Comment (Dmitrijg24k) [reply src/core/blockchain/data-loader.ts:197]: So, i need to send the network argument explicitly ( getItemProperty(postId, network: network1) )?

Comment (sergeyi23) [reply src/core/blockchain/data-loader.ts:197]: I see now that the remaining 2 are optional. All good then.

**Review by sergeyi23 — APPROVED** [review]

**Pull request merged**